packages/ui/src/components/va-date-input/VaDateInput.vue

```diff
@@ -239,7 +239,7 @@ const modelValueToString = (value: DateInputModelValue): string => {
     return ''
   }
 
-  throw new Error('VaDatePicker: Invalid model value. Value should be Date, Date[] or { start: Date, end: Date | null }, got ' + typeof value)
```

Comment:
There was error, because by default clearValue is empty string. `''`

Reply:
Ah okay gotcha. Thanks for the explanation and the fix!